TableTennisGenerator/TableTennisScorer/Program.cs

```diff
@@ -9,35 +10,24 @@ class Program
         public static void Main(string[] args)
         {
             string inputFile = "C:\\input\\sample_scoring_input.csv";
-            List<Tuple<string, string, string, string, int, int>> matches = ParseCSV(inputFile);
+            Scorer scorer = new Scorer(inputFile);
+            Dictionary<string, Dictionary<string, double>> metrics = scorer.GenerateMetrics();
 
-            foreach (Tuple<string, string, string, string, int, int> match in matches)
-            {
-                Console.WriteLine($"{match.Item1}+{match.Item2} vs {match.Item3}+{match.Item4} score={match.Item5} to {match.Item6}" );
-            }
-            Console.WriteLine("Hello World!");
-        }
 
-        public static List<Tuple<string, string, string, string, int, int>> ParseCSV(string path)
-        {
-            List<Tuple<string, string, string, string, int, int>> results = new List<Tuple<string, string, string, string, int, int>>();
-            using (StreamReader reader = new StreamReader(path))
+            string outputFile = "C:\\output\\sample_scoring_output.csv";

```

Comment:
what about putting this in a method on Scorer (``WriteOutputs`` or something) that takes an output file string name as a parameter?